rtengine/procparams.cc

```diff
@@ -3306,6 +3327,10 @@ int ProcParams::save(const Glib::ustring& fname, const Glib::ustring& fname2, bo
         saveToKeyfile(!pedited || pedited->hsvequalizer.scurve, "HSV Equalizer", "SCurve", hsvequalizer.scurve, keyFile);
         saveToKeyfile(!pedited || pedited->hsvequalizer.vcurve, "HSV Equalizer", "VCurve", hsvequalizer.vcurve, keyFile);
 
+// Soft Light
+        saveToKeyfile(!pedited || pedited->softlight.enabled, "SoftLight", "Enabled", softlight.enabled, keyFile);
+        saveToKeyfile(!pedited || pedited->softlight.strength, "SoftLight", "Strength", softlight.strength, keyFile);
+
```

Comment:
Here soft light is in front of film simulation. In most other places it's FS then SL. Would be nice and clean if you could streamline it.

Reply:
good catch. This is a leftover from when I was putting soft light before film simulation in the pipeline. I'll fix that, thanks for spotting!